themes/jaeger-docs/assets/sass/navbar.sass

```diff
@@ -14,4 +14,7 @@
 
       &.is-small
         padding: 0 0 0 2rem
-        font-size: 1.1rem
\ No newline at end of file
+        font-size: 1.1rem
+
+      &:hover
+        background-color: $navbar-item-hover-background-color
```

Comment:
`$navbar-item-hover-background-color` was already previously defined and the color **was** already changing to light grey. So there had to be a style defined that used that - why do we need a new one?

Reply:
The original way of writing would cause the background color of the dropdown item to be inconsistent when hovering because this line was absent.

But after looking into it again, I think I should change the `$navbar-dropdown-item-hover-background-color` variable instead of working on the CSS myself.